doc/source/rllib/rllib-training.rst

```diff
@@ -811,12 +812,23 @@ Tune will schedule the trials to run in parallel on your Ray cluster:
         metric="episode_reward_mean", mode="max"
     )
 
-Loading and restoring a trained agent from a checkpoint is simple:
+Loading and restoring a trained algorithm from a checkpoint is simple.
+For RLlib checkpoint versions of >= v1.0 (you can find your checkpoint's version by
+looking into the ``rllib_checkpoint.json`` file inside your checkpoint directory), do:
 
 .. code-block:: python
 
-    agent = ppo.PPO(config=config, env=env_class)
-    agent.restore(checkpoint_path)
+    from ray.rllib.algorithms.algorithm import Algorithm
+    algo = Algorithm.from_checkpoint(checkpoint_path)
+
+
+For older RLlib checkpoint versions (v0.1), you can still restore an algorithm via:
+
+.. code-block:: python
+
```

Comment:
do we want to add the import statement here, too?

Reply:
done